backend/django/core/views/api_admin.py

```diff
@@ -362,15 +362,15 @@ def get_project_status(request, project_pk):
     )
     unlabeled_data_objs = list(
         project.data_set.filter(datalabel__isnull=True)
-        .exclude(id__in=stuff_in_queue)
```

Comment:
Same question as recycled. By removing this exclude call, we add the admin items to this total.